### ISSUE TITLE
Suggestion: A way to color mods

### ISSUE THREAD

**Issue by Hope-Caldwell** [RimSort/RimSort#543]
Problem:
Difficulties in debugging large modlists 
I have an about 600 mod modlist, and it is currently very hard to single out a mod that is causing issues using rimsort because of the inability to distinguish mods quickly via color.

Solution:
Adding a way to color mods.
Adding a way to quickly and easily distinguish between mods.

**Comment by hk21702** [comment]
Blocked by #403 though likely will not in of itself be implemented in #403

**Comment by Limfjorden** [comment]
This issue is a duplicate of https://github.com/RimSort/RimSort/issues/109

**Comment by hk21702** [comment]
Closing since duplicate. Please continue discussions in relevant issue.
